templates/service_templates/redis.service.erb

```diff
@@ -3,6 +3,8 @@ Description=Advanced key-value store for <%= @title %>
 After=network.target
 
 [Service]
+RuntimeDirectory=redis
+RuntimeDirectoryMode=0755
```

Comment:
Could you please test this with ```RuntimeDirectoryMode=2755```? I have seen this at Debian and Ubuntu.

Reply:
Ok!
I'll give you the test results in a few minutes.

Reply:
Testing finished with Debian 10.2 and Redis package version 5.0.3-4+deb10u1

Work's like a charm!